webhooks/src/main/scala/zio/webhooks/backends/sttp/WebhookSttpClient.scala

```diff
@@ -0,0 +1,38 @@
+package zio.webhooks.backends.sttp
+
+import _root_.sttp.client._
+import _root_.sttp.client.asynchttpclient.zio.AsyncHttpClientZioBackend
+import _root_.sttp.model.Uri
+import zio._
+import zio.webhooks.WebhookHttpClient
+import zio.webhooks.WebhookHttpRequest
+import zio.webhooks.WebhookHttpResponse
+
+import java.io.IOException
+
+/**
+ * A [[WebhookSttpClient]] provides a [[WebhookHttpClient]] backend which is in turn backed by
+ * sttp's ZIO backend, specifically the [[AsyncHttpClientZioBackend]].
+ */
+final case class WebhookSttpClient(sttpClient: SttpClient) extends WebhookHttpClient {
+
+  // TODO: should we use `Uri.safeApply` here instead? Is it on this lib to ensure proper URIs?
+  // Right now we're failing fast when an invalid Uri is passed in.
```

Comment:
ZIO libraries should never fail eagerly. If they fail for defects, they should do so lazily. Wrap the whole function body in `effectSuspendTotal` to achieve this.